<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmokitServerUIClass</class>
 <widget class="QMainWindow" name="EmokitServerUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmokitServerUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="emokitserverui.qrc">
    <normaloff>:/EmokitServerUI/ServerIcon.ico</normaloff>:/EmokitServerUI/ServerIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>控制区</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="StopServer">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="StartServer">
        <property name="text">
         <string>启动</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="ClearMsgButton">
        <property name="text">
         <string>清空消息框</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>消息框</string>
    </property>
    <widget class="QTextBrowser" name="ShowInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>在线用户</string>
    </property>
    <widget class="QListView" name="UserListView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emokitserverui.qrc"/>
 </resources>
 <connections/>
</ui>
